### ISSUE TITLE
R Fundamentals

### ISSUE THREAD

**Issue by pgstevenson** [TelethonKids/RWorkshop#1]

**Comment by nzcoops** [comment]
- [x] Complete R more recently
- [x] Complete What R isn't
- [x] RStudio IDE fullscreen image
- [x] Finish reading data in
- [x] Finish installing/loading packages
- [x] Finish saving within R
- [x] Finish logical class
- [x] Finish factor class
- [x] Finish date class
- [x] Finish classes other
- [x] Finish structures list
- [x] Finish structures matrix/array
- [x] Finish structures tibble
- [x] Other basic syntax?
- [x] Update data to use Paul's health dataset
- [x] Tidy "## Reading data in - example" code wrapping / line breaks

**Comment by nzcoops** [comment]
Does this talk need a tricks/catches section? Or is that too big?

Things like variable names with spaces dat$`date of birth` etc?

I'm thinking not.

**Comment by pgstevenson** [comment]
> Does this talk need a tricks/catches section? Or is that too big?
> 
> Things like variable names with spaces dat$`date of birth` etc?
> 
> I'm thinking not.

I don't think so. If anything, we should discourage this for a "better" standard (all lower snake/camel case)

**Comment by nzcoops** [comment]
> I don't think so. If anything, we should discourage this for a "better" standard (all lower snake/camel case)

Haha, I see how you read that. What I meant was the use of ` IF you have a variable name with spaces (that's come in from some read/load (web/excel)). But I see the ` disappeared/turned it into code. 

But in general, I think a tips n tricks section would always be a bit too vague/cheery picking.

**Comment by nzcoops** [comment]
Slides finalised - passed over for review
